pulp_rpm/tests/functional/constants.py

```diff
@@ -1636,24 +1636,22 @@
         "static_context": None,
         "context": "6c81f848",
         "arch": "x86_64",
-        "artifacts": {
-            "rpms": [
-                "postgresql-0:9.6.8-1.module_1710+b535a823.x86_64",
-                "postgresql-0:9.6.8-1.module_1710+b535a823.src",
-                "postgresql-libs-0:9.6.8-1.module_1710+b535a823.x86_64",
-                "postgresql-server-0:9.6.8-1.module_1710+b535a823.x86_64",
-                "postgresql-test-0:9.6.8-1.module_1710+b535a823.x86_64",
-            ]
-        },
-        "dependencies": [
-            {"requires": {"platform": ["f29"]}, "buildrequires": {"platform": ["f29"]}}
+        "artifacts": [
```

Comment:
just making sure, `RPM_MODULEMDS_DATA` is not supposed to be valid modularity metadata as defined in https://github.com/fedora-modularity/libmodulemd/blob/main/yaml_specs/modulemd_stream_v2.yaml#L669, right?
that's the *pulp* representation of the data inside a module stream?

Reply:
A full copy of the yaml is stored inside the snippet, and that's the only thing used for reproduction of the metadata.  This is just a copy as stored by Pulp for the purposes of making queries against it.  This is why we can fix the metadata by just parsing the snippet again and re-saving it.